nodejs/webrtc-hello-video/public/index.html

```diff
@@ -0,0 +1,142 @@
+<link rel="icon" href="data:;base64,iVBORw0KGgo=" />
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<link href="main.css" rel="stylesheet" />
+<!-- please host this yourself, referencing github for the demo only -->
+<!-- <script src="https://raw.githubusercontent.com/Bandwidth/webrtc-browser/EDGE-602/bundle/0.5.2/BandwidthRtc.bundle.js"></script> -->
```

Comment:
I think this link would be preferred if you're going to reference it directly: https://github.com/Bandwidth/webrtc-browser/releases/download/0.5.2/BandwidthRtc.bundle.js

Reply:
It might be best to point to that and not include the bundle with the App, but I don't have strong opinions on this

Reply:
that was an old link thanks for catching that, going with the live link rather than the committed bundle.